lfucg-exactions/server/plats/utils.py

```diff
@@ -38,6 +24,8 @@ def calculate_lot_balance(lot_id):
         lot.dues_open_space_dev +
         lot.dues_open_space_own)
 
+    total_exactions = sewer_exactions + non_sewer_exactions
+
```

Comment:
I think we still need a matching block like the payments one below for account ledgers applied to the lot (to calculate two new variables: sewer_credits_applied & non_sewer_credits_applied).  These sums should be subtracted from the right hand side of the equations for current_exactions, sewer_due, and non_sewer_due.